### DISCUSSION TITLE
32bit vs 64bit Windows?

### DISCUSSION THREAD

**Discussion by forsythac** [jacktrip/jacktrip#721] [General]
Does JackTrip support 32bit Windows machines? Do we need different executables to support them?

**Comment by forsythac** [comment]
Looks like we install in 32bit directory on windows (x86) but with a 64bit app. seems like we should either:

- change the build to be 32bit (probably best? unless there's a reason to be 64bit)
- install in 64bit folder if we need to do 64bit

**Reply by dyfer** [reply]
Most software is 64-bit these days. We should definitely not switch the build to be 32-bit only. We should, however, install in the correct `Program Files` directory.

If needed, we could consider creating an _additional_ 32-bit build, per discussion in #725

**Reply by psiborg112** [reply]
#750 now includes changes to install into the 64 bit program files folder.
